tests/mixins/test_search.py

```diff
@@ -115,3 +115,27 @@ def test_search_library(self, config, yt_oauth):
             yt_oauth.search("beatles", filter="community_playlists", scope="library", limit=40)
         with pytest.raises(Exception):
             yt_oauth.search("beatles", filter="featured_playlists", scope="library", limit=40)
+
+    def test_remove_suggestion_valid(self, yt_auth):
+        first_pass = yt_auth.search("b")
+        assert len(first_pass) > 0, "Search returned no results"
```

Comment:
I guess you're only calling `search` here to populate the history first? Could you add a comment stating this?